src/emcpy/plots/variable_specs.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+
+vardict = {
+    'temperature': {
+        'name': 'temperature',
+        'short name': 't',
+        'variable type': 'conventional',
+        'units': 'K',
+        'range': [220, 320],
+        'contour interval': 5,
+        'diff range': [-5, 5],
+        'diff contour interval': 0.5,
+        'cmap': 'rainbow'
+    },
+    'specific humidity': {
+        'name': 'specific humidity',
+        'short name': 'q',
+        'variable type': 'conventional',
+        'units': 'kg/kg',
+        'range': [0, 0.030],
+        'contour interval': 0.0025,
+        'diff range': [-0.005, 0.005],
+        'diff contour interval': 0.001,
+        'cmap': 'YlGnBu'
+    },
+    'ugrd': {
+        'name': 'horizontal wind',
+        'short name': 'ugrd',
+        'variable type': 'conventional',
+        'units': 'm/s',
+        'range': [-50, 50],
+        'contour interval': 5,
+        'diff range': [-5, 5],
+        'diff contour interval': 0.5,
+        'cmap': 'PuOr'
+    },
+    'vgrd': {
+        'name': 'meridional wind',
+        'short name': 'vgrd',
+        'variable type': 'conventional',
+        'units': 'm/s',
+        'range': [-50, 50],
+        'contour interval': 5,
+        'diff range': [-5, 5],
+        'diff contour interval': 0.5,
+        'cmap': 'PuOr'
+    },
+    'wind speed': {
+        'name': 'wind speed',
+        'short name': 'wndspd',
+        'variable type': 'conventional',
+        'units': 'm/s',
+        'range': [0, 75],
+        'contour interval': 5,
+        'diff range': [-5, 5],
+        'diff contour interval': 0.5,
+        'cmap': 'viridis'
+    },
+    'brightness temperature': {
+        'name': 'brightness temperature',
+        'short name': 'bt',
+        'variable type': 'radiance',
+        'units': 'K',
+        'range': [220, 320],
+        'contour interval': 5,
+        'diff range': [-5, 5],
+        'diff contour inte rval': 0.5,
+        'cmap': 'rainbow'
+    }
+}
```

Comment:
Can we put this in a yaml?
And load the yaml from a `cfg` directory?
Don't need to come here to make changes if someone wants to change colormap or range, etc.

Reply:
I like this suggestion, but suggest that we do this at a later date (create an issue in the repo to add this functionality) as that will introduce a `yaml` dependency for `emcpy`

Reply:
This is fine for now.

Reply:
make an issue and mark it to do later.